src/PresentationalComponents/WithPermission/WithPermission.js

```diff
@@ -0,0 +1,20 @@
+import React  from 'react';
```

Comment:
Nice to have later: This component together with PermissionRouter can be pulled into FEC as shared and then used across applications.